package-lock.json

```diff
@@ -4774,9 +4774,9 @@
       "dev": true
     },
     "needle": {
-      "version": "2.3.2",
-      "resolved": "https://registry.npmjs.org/needle/-/needle-2.3.2.tgz",
-      "integrity": "sha512-DUzITvPVDUy6vczKKYTnWc/pBZ0EnjMJnQ3y+Jo5zfKFimJs7S3HFCxCRZYB9FUZcrzUQr3WsmvZgddMEIZv6w==",
+      "version": "2.5.0",
```

Comment:
I upgraded the deps that node-mapnik uses since the old needle version was causing an install failure on node v12. This fixed it.